nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -197,9 +197,141 @@ def test_plot_surf_roi_error():
     mesh = generate_surf()
     rng = np.random.RandomState(0)
     roi_idx = rng.randint(0, mesh[0].shape[0], size=5)
-
-    # Wrong input
     with pytest.raises(
             ValueError,
             match='roi_map does not have the same number of vertices'):
         plot_surf_roi(mesh, roi_map=roi_idx)
+
+
+def _generate_img():
```

Comment:
is there a function in `_utils.data_gen` that you could use here?

Reply:
I was thinking maybe `generate_fake_fmri`or `generate_maps` but if they don't fit the job that's fine